autopush/tests/test_websocket.py

```diff
@@ -54,6 +54,7 @@ def setUp(self):
         self.proto.sendClose = self.close_mock = Mock()
         self.proto.transport = self.transport_mock = Mock()
         self.proto.closeHandshakeTimeout = 0
+        self.proto.autoPingInterval = 300
```

Comment:
Removing my concern after discussion. We can derive the information from analyzing the logging metadata. The metric doesn't carry enough data to be useful.